llms/mlx_lm/utils.py

```diff
@@ -126,7 +161,9 @@ def generate(
     model: nn.Module,
     tokenizer: PreTrainedTokenizer,
     prompt: str,
-    temp: float = 0.0,
+    temp: Optional[float] = 0.0,
+    repetition_penalty: Optional[float] = 1.0,
+    repetition_context_size: Optional[int] = 5,
```

Comment:
Nit: Set the default repetition_context_size to 0 in order to avoid unnecessarily saving the token context if the user doesn't use `repetition_penalty`.